components/integrations/bots/index.js

```diff
@@ -4,20 +4,25 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 import {getBotAccounts} from 'mattermost-redux/selectors/entities/bots';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {loadBots, disableBot, enableBot} from 'mattermost-redux/actions/bots';
 import {createUserAccessToken, revokeUserAccessToken, enableUserAccessToken, disableUserAccessToken, getUserAccessTokensForUser, getUser} from 'mattermost-redux/actions/users';
 import * as UserSelectors from 'mattermost-redux/selectors/entities/users';
 
 import Bots from './bots.jsx';
+import { create } from 'domain';
```

Comment:
Was this an intentional change?

Reply:
No, it's not. I am bit a bit confused how that got there but I'll change it.